data_utils.py

```diff
@@ -0,0 +1,5 @@
+import torchaudio
+
+def load_wav(path):
+    from glob import glob
+    return list(map(torchaudio.load, glob('*.wav')))
```

Comment:
sorted(glob('*.wav')) might be better. The outputs of glob are sometimes not sorted well.

Reply:
I don't use this function now, I'll delete it